everest.yaml

```diff
@@ -3,4 +3,6 @@
   DLL: bin/Debug/SkinModHelper.dll
   Dependencies:
     - Name: Everest
-      Version: 1.2742.0
\ No newline at end of file
+      Version: 1.2742.0
+    - Name: CelesteNet.Client
```

Comment:
I definitely think CN should be an optional dependency, so this will need some logic to only run the CN logic if the dependency is loaded

Reply:
Yep, I gotta add some more checkboxes to the first post